snapcraft/__init__.py

```diff
@@ -377,6 +377,22 @@ def find_library(name):
     import ctypes.util
     ctypes.util.find_library = find_library
 
+
+def _get_version():
+    if _os.environ.get('SNAP_NAME') == 'snapcraft':
+        with open(_os.path.join(_os.getenv('SNAP'), 'meta', 'snap.yaml')) as f:
+            snap_yaml = yaml.load(f)
+        return snap_yaml['version']
```

Comment:
I know you didn't write this originally, but is there a reason we're not simply using `SNAP_VERSION` here as opposed to the yaml dance?